src/main/webapp/app/exam/manage/students/upload-images/students-upload-images-dialog.component.scss

```diff
@@ -0,0 +1,107 @@
+$success-text-color: #155724;
```

Comment:
No color definitions outside the theme file!

Reply:
@pellumbbaboci please add this in a follow-up PR. You can contact @bassner if you need help!